javascript/ql/test/library-tests/FlowLabels/DefaulFlowLabels.ql

```diff
@@ -0,0 +1,6 @@
+import javascript
+
+// Check which flow labels are materialize by importing `javascript.qll`.
+// If this increases, it may indicate a performance issue.
+
+select any(DataFlow::FlowLabel label)
```

Comment:
```suggestion
// Check which flow labels are materialized by importing `javascript.qll`.
// If this increases, it may indicate a performance issue.
select any(DataFlow::FlowLabel label)

```

1) `materialize` -> `materialized`. 
2) autoformat.